src/main/resources/standardTestCases/refresh_token.test

```diff
@@ -1,5 +1,6 @@
 name: refresh_token
 testFormat: workflowsuccess
+backend: JES
```

Comment:
this might have to be "jes-refresh" as that's what the backend is called in the .conf